test/testinput/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,169 @@
+#(C) Copyright 2021 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# SAID, CLAT, CLON, YEAR, MNTH, DAYS, HOUR, MINU, SECO, SWCM, SAZA, OGCE, SCCF, SWQM
+# HAMD, PRLC, WDIR, WSPD
+# GNAP, PCCF
+
+observations:
+  - obs space:
+      name: bufr
+      #obsdatain: "./testinput/bufr_read_2_dim_blocks.bufr"
+      obsdatain: "./testinput/bufr_ncep_satwind_avhrr.bufr_d"
+
+      exports:
+        variables:
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          latitude:
+            query: "*/CLAT"
+
+          longitude:
+            query: "*/CLON"
+
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+
+          satelliteDerivedWindComputationMethod:
+            query: "*/SWCM"
+
+          satelliteZenithAngle:
+            query: "*/SAZA"
+
+          originatingCenter:
+            query: "*/OGCE"
+
+          satelliteChannelCentreFrequency:
+            query: "*/SCCF"
+
+          sdmeditSatWindQualMark:
+            query: "*/SWQM"
+
+          heightAssignmentMethod:
+            query: "*/HAMD"
+
+          pressure:
+            query: "*/PRLC"
+
+          windDirection:
+            query: "*/WDIR"
+
+          windSpeed:
+            query: "*/WSPD"
+
+          windSpeedGeneratingApplication:
+            query: "*/GQCPRMS/GNAP[1]"
+
+          windDirectionGeneratingApplication:
+            query: "*/GQCPRMS/GNAP[2]"
+
+          windSpeedPercentConfidence:
+            query: "*/GQCPRMS/PCCF[1]"
+
+          windDirectionPercentConfidence:
+            query: "*/GQCPRMS/PCCF[2]"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/bufr_ncep_satwind_avhrr.nc"
+      dimensions:
+        - name: Confidence
+          paths:
+            - "*/GQCPRMS"
+
+      variables:
+        - name: "MetaData/satelliteIdentifier"
+          source: variables/satelliteIdentifier
+          longName: "Satellite Identifier"
+          units: ""
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude (Coarse Accuracy)"
+          units: "degree_north"
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude (Coarse Accuracy)"
+          units: "degree_east"
+
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/satelliteDerivedWindComputationMethod"
```

Comment:
Change this to "MetaData/windComputationMethod"

(This is from the [JEDI convention spreadsheet](https://docs.google.com/spreadsheets/d/1fHb2QIHOwgp4d6plyn9Dj9HZFZsDRxcPaUV6hdTb5BU/edit#gid=0))

Reply:
The yaml has been updated.
```
          windComputationMethod:
            query: "*/SWCM"

        - name: "MetaData/windComputationMethod"
          source: variables/windComputationMethod
          longName: "Satellite Derived Wind Computation Method"
          units: ""
```